test/service/test_request_validation.py

```diff
@@ -29,6 +29,14 @@ def lambda_name(cls) -> str:
                           "Message": "BadRequestError: Unable to sort by undefined facet bad-facet."}
     service_config_dir = os.path.dirname(service_config.__file__)
 
+    def _bad_query_message(self, key, type_=None):
```

Comment:
_Consider_ making a `@classmethod`.

Reply:
Also, I know @hannes-ucsc often prefers verbatim assertions in tests instead of computed results, though for this case I doubt he'd object.